reporters_db/data/reporters.json

```diff
@@ -18172,35 +18172,40 @@
                 "Ridgely's Notebook": {
                     "end": "1830-12-31T00:00:00",
                     "regexes": [
-                        "$volume_nominative ?$reporter, $page"
+                        "$volume_nominative ?$reporter, $page",
+                        "$volume_nominative ?(?P<reporter>Ridgely(.)s Notebook\\s?(I)?), $page"
```

Comment:
this is strange no?  why do we need to this?

Reply:
In this case we can have Ridgely's Notebook I or Ridgely's Notebook.

With unicode character we have same variations, the regex "$volume_nominative ?(?P<reporter>Ridgely(.)s Notebook\\s?(I)?), $page" handle the characters, i updated it to make it more explicit, to this: "$volume_nominative ?(?P<reporter>Ridgely(’|')s Notebook\\s?(I)?), $page" only alllows apostrophe or unicode quotation mark